repo/packages/A/avi/3/config.json

```diff
@@ -0,0 +1,207 @@
+{
```

Comment:
[dcos.zip](https://github.com/mesosphere/universe/files/1049962/dcos.zip)
attached a zip'ed mp4 video showing our default deployment on v1.9.
Maybe you are hitting the resource constraints? Do you have atleast 1 node in '*' role with sufficient resources - default is 4cpu, 12G ram, 64G disk.